drake-torch-melodic.dockerfile

```diff
@@ -71,11 +71,16 @@ RUN cd $HOME/googletest-release-1.10.0/build \
     && rm -rf googletest-release-1.10.0
 
 # boost 1.74 without removing libboost 1.65 on which ROS depends
-RUN curl -SL https://dl.bintray.com/boostorg/release/1.74.0/source/boost_1_74_0.tar.bz2 | tar -xj \
-    && cd boost_1_74_0 \
-    && ./bootstrap.sh --prefix=/usr --with-python=python3 \
-    && ./b2 stage -j 12 threading=multi link=shared \
-    && ./b2 install threading=multi link=shared
+# RUN curl -SL https://dl.bintray.com/boostorg/release/1.74.0/source/boost_1_74_0.tar.bz2 | tar -xj \
```

Comment:
add a comment explaining or remove?

Reply:
it's in the commit message that can be viewed by git blame

Reply:
- the message seemed tentative, could you add a comment as to why commenting it out makes it work? 

- is boost 1.65 all that remains now?

- would we want to consider uncommenting it again in the future?

Reply:
i only have suspicions as to why it works. installing 1.74 overrides the headers in `/usr/include` and `cv_bridge melodic` may be getting segfault due to mixing 1.65 so and 1.74 headers. we could install to ``/usr/include/boost-1.74` like OMPL does by default but it's probably not worth it. if we need 1.74 we'll do that.

the current nightly channel has focal and 1.71 which isn't very recent either but probably good enough.

Reply:
fair enough, I think it would be best to have this documented as a comment in the dockerfile itself for when someone else stumbles upon this block of code in a month or two :+1: